src/components/Facets/Facets.tsx

```diff
@@ -91,6 +88,9 @@ export const Facets: FunctionComponent<FacetsProps> = ({
 }: FacetsProps) => {
   const {config} = useStore();
   const translation = useTranslation();
+
+  searchFacets = searchFacets.filter(facet => facet.buckets.length > 0);
+
   const searchFacetsSliced = searchFacets?.slice(0, 3)
```

Comment:
Is this slice impacted by filtering searchFacets ?
I'll approve by make sure it is working for other sites as well, like wilson.

Reply:
It is, but that's the idea. Slice is to have only first three facets as buttons that activate popup with filters. Filtering facets was to make sure that there are no links that are not present in filters themselves cause this buttons also are opening filter tabs etc. So without filtering it was creating issues like opening wrong filter for example.

But I will check all sites again to make sure it does not create issues.